### PR TITLE
Add Russian translation and Polish

### PR DESCRIPTION
**Pull request by DenisMihnuk**
In the futur me cousin he make a Polish translate

### CODE REVIEW

**Comment by SkyWarZzeur** [comment]
Hi! can you add the changes that were made in this commit f3c9ed4dda3d20ce9daa0ceff083b5c7631fa2c3 please? When it's good I'll merge it.

**Comment by SkyWarZzeur** [comment]
Another review is needed.

**Comment by SkyWarZzeur** [comment]
Hi, the translation of this commit 42c6e1e57dfe444f1df8448b6c77560e152e8fef is needed too!

File: docs-pl/plugins.md [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+```html
+@extends('layouts.app')
+
+@section('title', 'Nazwa strony')
+
+@section('content')
+    <div class="container content">
+        <h1>Un titre</h1>
+
+        <p>Un texte</p>
```
Comment (SkyWarZzeur) [inline docs-pl/plugins.md]: Missing translation

File: docs-ru/plugins.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+```html
+@extends('layouts.app')
+
+@section('title', 'Nom de la page')
+
+@section('content')
+    <div class="container content">
+        <h1>Un titre</h1>
+
+        <p>Un texte</p>
```
Comment (SkyWarZzeur) [inline docs-ru/plugins.md]: Missing translation

File: docs-ru/plugins.md [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+Представления являются видимой частью плагина, они являются файлами содержимого HTML
+плагина для отображения страницы.
+
+В Azuriom, использующим представления [Laravel](https://laravel.com/), представления могуть быть сделаны при помощи Blade.
+Если Вы еще не владеете Blade, то мы настоятельно рекомендуем ознакомиться с информацией по ссылке [its documentation](https://laravel.com/docs/6.x/blade).
+
+> {warn} Настоятельно рекомендуется НЕ ИСПОЛЬЗОВАТЬ синтаксис PHP
+при работе с Blade, поскольку это лишь усложнит дальнейшую работу.
+
+Для отображения представления, используйте `view('<plugin slug>::<название представления>')`,
```
Comment (SkyWarZzeur) [inline docs-ru/plugins.md]: Missing translation

File: docs-pl/plugins.md [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+Widoki są widoczną częścią wtyczki; są to pliki treści HTML
+wtyczki do wyświetlenia strony.
+
+W Azuriom przy użyciu widoków [Laravel](https://laravel.com/) widoki można tworzyć za pomocą Blade.
+Jeśli nie masz jeszcze Blade'a, zdecydowanie zalecamy przeczytanie informacji na [jego dokumentacji](https://laravel.com/docs/6.x/blade).
+
+> {warn} Zdecydowanie zaleca się NIE używać składni PHP
+podczas pracy z Blade, ponieważ to tylko skomplikuje dalszą pracę.
+
+Aby wyświetlić widok, użyj `view('<plugin slug>::<view name>')`,
```
Comment (SkyWarZzeur) [inline docs-pl/plugins.md]: Missing translation

**Comment by MrMicky-FR** [comment]
Closing for inactivity and the translations of the documentation will be done on [Crowdin](https://crowdin.com/project/azuriom).

**Pull request closed without merging**